variable.go

```diff
@@ -433,11 +434,33 @@ func (vr *variableResolver) resolve(ctx *ExecutionContext) (*Value, error) {
 				currArgs = append([]functionCallArgument{executionCtxEval{}}, currArgs...)
 			}
 
+			// If kwargs map is needed
+			includeKwargsBit := 0
+			if t.NumIn() > 0 && t.In(0) == typeOfKwargsMap {
+				includeKwargsBit = 1
+			}
+
+			evaluatedArgs := make([]*Value, 0)
+			kwargCount := 0
+			for _, arg := range currArgs {
+				v, err := arg.Evaluate(ctx)
+				if err != nil {
+					return nil, err
+				}
+				if v.IsKwarg() {
+					if !(includeKwargsBit == 1) {
+						return nil, fmt.Errorf("calling a function using a keyword argument: %v=%v, but the function does not support kwargs", v.Name(), v)
+					}
+					kwargCount++
+				}
+				evaluatedArgs = append(evaluatedArgs, v)
+			}
+
 			// Input arguments
-			if len(currArgs) != t.NumIn() && !(len(currArgs) >= t.NumIn()-1 && t.IsVariadic()) {
+			if len(currArgs)-kwargCount != t.NumIn()-includeKwargsBit && !(len(currArgs)-kwargCount >= t.NumIn()-1-includeKwargsBit && t.IsVariadic()) {
```

Comment:
Should add a test for this also.

Reply:
https://github.com/rudderlabs/pongo2/pull/16/files#diff-bdd07de42682b40d2719285bebc2e1c75da43541c42861dd3b21c7c781076e82R309